autobrowser/tabs/crawlerTab.py

```diff
@@ -248,6 +251,16 @@ def main_frame_getter(self) -> Frame:
         await self.redis.lpush(self.config.redis_keys.auto_done, end_info)
         await super().close()
 
+    def memento_header_to_timestamp(self, memento_dt):
```

Comment:
Perhaps it would be simplest to move the logic of setting the `_timestamp` instance property into this function

```suggestion
    def timestamp_from_response(self, response: Response) -> None:
        memento_dt = response.headers.get("Memento-Datetime")
        ....
        self._timestamp = dt.strftime('%Y%m%d%H%M%S')
        ....
```

Otherwise I think this function is best home is Helper with signature
```python
def memento_header_to_timestamp(memento_dt: Optional[str]) -> str:
```

Since it is likely the timestamp retrieved by `BaseTab.post_behavior_run` should also be converted (unless I have forgotten the format of `window.wbinfo.timestamp`)

Also could you add a quick doc string to method.

Reply:
Renamed to set_timestamp_from_response as suggested and added docstring.
Removed checking for wbinfo to avoid dependency on wombat (which should set Memento-Datetime anyway). If no Memento-Datetime, just use current UTC timestamp.